include/kernels/dslash_wilson2.cuh

```diff
@@ -0,0 +1,249 @@
+#pragma once
```

Comment:
Why do we need this file?

Reply:
Accidental addition.  Can fix when I’m back 💻

Reply:
I just removed it.